ng build/serve fails when importing modules from igniteui-angular (latest patch -14.0.5)

## Description  
ng build/serve fails when importing modules from igniteui-angular (latest patch -14.0.5)

 * igniteui-angular version: 14.0.5
 * browser: -

## Steps to reproduce  
Angular CLI: 14.0.4

`ng new testProject`
`cd testProject`
`ng add @infragistics/igniteui-angular@14.0.5`
`ng build`

_running ng build at this point results in successful build_

either
Add new component template with IG CLI:
`ig add grid newGrid`

or simply add any igniteui-angular module in app.module.ts imports array, for example IgxIconModule

`import { IgxIconModule } from '@infragistics/igniteui-angular';`
```
@NgModule({
 /*...*/
  imports: [
    BrowserModule,
    IgxIconModule
  ],
  /*...*/
})
```

Run `ng build` or `ng serve`

## Result  
$ ng build
✔ Browser application bundle generation complete.

Error: node_modules/@infragistics/igniteui-angular/lib/grids/hierarchical-grid/hierarchical-grid.component.d.ts:184:48 - error TS2307: Cannot find module 'igniteui-angular' or its corresponding type declarations.

184     get excelStyleFilteringComponent(): import("igniteui-angular").IgxGridExcelStyleFilteringComponent;


## Expected result  
Successful build.

Not reproducible for igniteui-angular@14.0.4 or prior.

--- a/projects/igniteui-angular/src/lib/grids/hierarchical-grid/hierarchical-grid.component.ts
+++ b/projects/igniteui-angular/src/lib/grids/hierarchical-grid/hierarchical-grid.component.ts
@@ -42,6 +42,7 @@ import { IgxPaginatorComponent } from '../../paginator/paginator.component';
 import { IgxGridComponent } from '../grid/grid.component';
 import { IgxOverlayOutletDirective } from '../../directives/toggle/toggle.directive';
 import { IgxColumnResizingService } from '../resizing/resizing.service';
+import { IgxGridExcelStyleFilteringComponent } from '../filtering/excel-style/grid.excel-style-filtering.component';
 
 let NEXT_ID = 0;
 
@@ -395,7 +396,7 @@ export class IgxHierarchicalGridComponent extends IgxHierarchicalGridBaseDirecti
     }
 
     /** @hidden @internal */
-    public get excelStyleFilteringComponent() {
+    public get excelStyleFilteringComponent() : IgxGridExcelStyleFilteringComponent {
         return this.parentIsland ?
             this.parentIsland.excelStyleFilteringComponents.first :
             super.excelStyleFilteringComponent;

